cmd/minishift/cmd/root.go

```diff
@@ -28,10 +28,12 @@ import (
 	openShiftCmd "github.com/minishift/minishift/cmd/minishift/cmd/openshift"
 	"github.com/minishift/minishift/pkg/minikube/config"
 	"github.com/minishift/minishift/pkg/minikube/constants"
+	"github.com/minishift/minishift/pkg/minishift/instancedata"
```

Comment:
what's about _config_ as package name instead of _instancedata_. Sounds cumbersome.